storybook/pages/MyProfilePreviewPage.qml

```diff
@@ -0,0 +1,189 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+
+import AppLayouts.Profile.views 1.0
+import AppLayouts.Profile.stores 1.0
+
+import Storybook 1.0
+
+import utils 1.0
+
+SplitView {
+    Logs { id: logs }
+
+    SplitView {
+        orientation: Qt.Vertical
+        SplitView.fillWidth: true
+
+        MyProfilePreview {
+            SplitView.fillWidth: true
+            SplitView.fillHeight: true
+
+            contactsStore: ContactsStore {
```

Comment:
As Michal pointed out in my PR here https://github.com/status-im/status-desktop/pull/8010#pullrequestreview-1152372653 , you should probably use a stub instead. It really made it simpler.